tools/img_split_dota.py

```diff
@@ -0,0 +1,363 @@
+import BboxToolkit as bt
```

Comment:
If possible, I suggest merging this function directly into img_split.py

Reply:
The `classes` will be output from `load_func`. You can pass it into `single_split`.

By the way. Creating a new file is also good to understand. I accept both modifications.

It's very appreciated of you to contribute to this repo.

Reply:
oh! I miss the function single_split.
nextime I will try to merge directly. 

Thanks.